rust-mode-tests.el

```diff
@@ -1555,8 +1594,9 @@ this_is_not_a_string();)"
   (rust-test-font-lock
    "let union = 7; union foo { x: &'union bar }"
    '("let" font-lock-keyword-face
-     ;; Ignore the first union, it's an unhighlighted variable.
+     ;; Ignore the first union, it's a variable.
```

Comment:
I think the comment should change, because the first occurrence of union is no longer being ignored.